examples/js/nextjs-new/README.md

```diff
@@ -0,0 +1,53 @@
+# Next.js
+
+This is an example project showing how to use the universal `@bugsnag/js` notifier to track server, browser, and vercel serverless errors on a Next.js project.
```

Comment:
Is "universal" a term we generally use?  Might be better to remove it to avoid any confusion with e.g. Universal Angular.

Reply:
That wording was taken from the old example but it's not a particularly useful term. I'll remove it.